internal/trace/traceutil.go

```diff
@@ -43,11 +44,15 @@ func IDFromSpan(span opentracing.Span) string {
 }
 
 // URL returns a trace URL for the given trace ID at the given external URL.
-func URL(traceID, externalURL string) string {
+func URL(traceID, externalURL, traceProvider string) string {
 	if traceID == "" {
 		return ""
 	}
 
+	if traceProvider == string(tracer.Datadog) {
+		return "https://app.datadoghq.com/apm/trace/" + traceID
+	}
```

Comment:
nit: Could we do the other way around, which looks more strongly typed?
```suggestion
	if tracerType(traceProvider) == tracer.Datadog {
		return "https://app.datadoghq.com/apm/trace/" + traceID
	}
```